lib/ansible/modules/cloud/amazon/aws_caller_facts.py

```diff
@@ -73,13 +78,25 @@ def main():
     try:
         caller_identity = client.get_caller_identity()
```

Comment:
I think it would be best to rename this variable to caller_facts as we're now extending it's meaning to be more than strictly the caller identity.

I like the approach you've taken of just adding the account alias information into the existing dict, but just think we need to rename it since it's purpose is changed by this commit.